### ISSUE TITLE
Small noisy patches in output

### ISSUE THREAD

**Issue by luser350** [NVIDIA/pix2pixHD#339]
Hi, I am training a face-retouching model using FFHQ and FFHQR datasets. Since I am training on 1024x1024 my batch size is 2. Here is the sample output 
![epoch008_synthesized_image](https://github.com/NVIDIA/pix2pixHD/assets/168648774/52c0ab6f-3711-4ecd-b177-f5abc0409b29)

As you can see there is a small noisy patch on the forehead of the image. This noisy patch consistently shows up on every generated image.
How can this problem be rectified @aaronp24  @lukeyeager @javl @aflat ?
[opt.txt](https://github.com/NVIDIA/pix2pixHD/files/15282284/opt.txt)

**Comment by luser350** [comment]
3 weeks and still no answer.......??????????

**Comment by zhangxiaojuan66** [comment]
你好，邮件已收到，祝你万事如意，生活愉快！

**Comment by luser350** [comment]
> Hello, I have received your email. I wish you all the best and a happy life!

Thanks for your reply, can you please answer my question? Best wishes alone don't help.

**Comment by javl** [comment]
@luser350 I haven't used pix2pix in a long time so I can't help you. 

Keep in mind: you're using a free and open source tool, nobody is obligated to help and comments filled with question marks or sneers toward other users aren't going to convince anyone to spend time on this issue. 
Maybe start by explaining what you've already tried yourself?

**Comment by luser350** [comment]
@javl thanks for your reply
Firstly I felt mocked by @zhangxiaojuan66 reply. I have updated my previous comment and answered your question

> what you've already tried yourself?

**Comment by Roshan-digi5** [comment]
Hello,
Im having the same issue in my case at top right corner i get a small patch of noisy pixels

**Comment by zhangxiaojuan66** [comment]
你好，邮件已收到，祝你万事如意，生活愉快！

**Comment by pyphan1** [comment]
@luser350 this is often caused by a high learning rate or a very low batchsize, it is an area of confusion where gradients must have exploded below 0 or higher than 255 in the region, surrounding channels often explode too but leave 1 or 2 channels at max, thus making the red, green and blue areas surrounding the black hole and it looks like a checkerboard or squares because it is caused by conv layers, try training a 512*512 model with a higher batch size and lower learning rate first to see if your training strategy will even give good results, if so then go ahead and train the 1024*1024 model which will use 4x more VRAM

**Comment by zhangxiaojuan66** [comment]
你好，邮件已收到，祝你万事如意，生活愉快！
